server/repository/src/db_diesel/master_list_line_row.rs

```diff
@@ -20,12 +21,13 @@ joinable!(master_list_line -> item_link (item_link_id));
 allow_tables_to_appear_in_same_query!(master_list_line, item_link);
 allow_tables_to_appear_in_same_query!(master_list_line, name_link);
 
-#[derive(Clone, Insertable, Queryable, Debug, PartialEq, Eq, AsChangeset)]
+#[derive(Clone, Insertable, Queryable, Debug, Default, PartialEq, AsChangeset)]
 #[diesel(table_name = master_list_line)]
 pub struct MasterListLineRow {
     pub id: String,
     pub item_link_id: String,
     pub master_list_id: String,
+    pub price: Option<f64>,
```

Comment:
A thought to hash out before going fully live - there is currently ambiguity around what exactly price is in the current work, it'd be nice to narrow it down. The main system that uses this field already uses it as "Unit price", and as far as I'm aware that's about the only reasonable thing to do - unless this field was a collection of `packsize : price` pairs.

Reply:
good point, and  it is - so maybe a rename to `unit_price` or `price_per_unit` 🤷